gslib/command_runner.py

```diff
@@ -172,7 +172,8 @@ def __init__(self, bucket_storage_uri_class=BucketStorageUri,
     else:
       self.command_map = self._LoadCommandMap()
 
-  def _LoadCommandMap(self):
+  @staticmethod
```

Comment:
http://google.github.io/styleguide/pyguide.html#217-function-and-method-decorators

Why do we need the @staticmethod decorators? The style guide discourages them.

It looks like all usages of this function are via `self._LoadCommandMap()`, so we can just add self as a parameter.